projects/go-lib/src/lib/components/go-select/go-select.component.scss

```diff
@@ -141,3 +141,42 @@
 body > .ng-dropdown-panel {
   margin-left: 4px;
 }
+
+.ng-dropdown-panel .ng-dropdown-panel-items .ng-option {
+  background: $theme-light-bg-active;
+  font-size: .875rem;
+
+  &.ng-option-marked,
+  &.ng-option-selected,
+  &.ng-option-selected.ng-option-marked {
+    background-color: $theme-light-bg-hover;
+
+    .ng-option-label {
+      font-weight: $weight-light;
+    }
+  }
+}
```

Comment:
Did you ever hear back regarding your comment on the Issue? Based on the way I interpret her original wording together with what I see in the DSM, I'm thinking this should maybe be something like this:
```css
.ng-dropdown-panel .ng-dropdown-panel-items .ng-option {
  font-size: .875rem;

  &.ng-option-marked {
    background: $theme-light-bg-active
  }

  &.ng-option-selected,
  &.ng-option-selected.ng-option-marked {
    background-color: $theme-light-bg-hover;

    .ng-option-label {
      font-weight: $weight-light;
    }
  }
}
```
In other words, I understand her as wanting the background of the options to be white, hovered options to have a light-gray background, and the selected option to have a slightly darker gray background, if that makes sense.